WidgetEmbedding/EmbedWidgetswithSSOusingToken/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmbedWidgetswithSSOusingToken
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            // Variable declaration to form the embed URL 
+            var dashboardUrl = "https://onpremise-demo.boldbi.com/bi/site/site2/dashboards/9b5e9121-1379-4724-afba-c9674eb4f867/Education/adventure%20datasource%20dashboard?isWidgetMode=true&WidgetId=32ed09f7-49ef-4468-9c56-ccc376dbca8a&"; // URL of the widget to be embedded. Add '? or &' at the end of the URL based on your widget URL.
```

Comment:
Consider the points mentioned in Dashboard PR.
![image](https://github.com/boldbi/iframe-token-based-authorization-sample/assets/92368346/e0787970-b65d-49c7-a43a-7f034b893cee)